File: Examples/Conversations/Obj-C/ConversationsExample/MyAnswerTableViewCell.m
//
//  MyAnswerTableViewCell.m
//  ConversationsExample
//
//  Copyright © 2016 Bazaarvoice. All rights reserved.
//

#import "MyAnswerTableViewCell.h"

@interface MyAnswerTableViewCell()

@property (weak, nonatomic) IBOutlet UILabel *answerTestLabel;

@end

@implementation MyAnswerTableViewCell


- (void)setAnswer:(BVAnswer *)answer{
    
    super.answer = answer;
    self.answerTestLabel.text = answer.answerText;
    
}

@end

File: Examples/Conversations/Swift/ConversationsExample/MyAnswerTableViewCell.swift
//
//  MyAnswerTableViewCell.swift
//  Conversations
//
//  Copyright © 2016 Bazaarvoice. All rights reserved.
//

import UIKit
import BVSDK

class MyAnswerTableViewCell: BVAnswerTableViewCell {
  
  @IBOutlet weak var answerTestLabel : UILabel!
  
  override var answer: BVAnswer? {
    didSet {
      answerTestLabel.text = answer?.answerText
    }
  }
  
  
}
